playground/mocks/thirdparty/duo-mock.js

```diff
@@ -0,0 +1,14 @@
+module.exports = {
+  init: (options) => {
+    // Load a mock iframe
+    // and add an event listener to continue with authentication
+    options.iframe.src = '/mocks/thirdparty/duo-iframe.html';
+    options.iframe.onload = () => {
```

Comment:
As a follow-up, we should consider adding a test with a loading delay/failure to validate how the widget behaves with these scenarios.

Reply:
https://oktainc.atlassian.net/browse/OKTA-369010

We have a try/tach now around `Duo.init`, so maybe we are covered. In any case, I created that follow up jira, please add comments there if I missed something